autotag/main.go

```diff
@@ -28,7 +28,7 @@ var opts Options
 func init() {
 	_, err := flags.Parse(&opts)
 	if err != nil {
-		log.Println(err)
+		fmt.Fprintf(os.Stderr, err.Error())
```

Comment:
Doesn't the standard logger send its output to `stderr` by default?
https://github.com/golang/go/blob/go1.17.3/src/log/log.go#L76

If so, then I think we should change any `fmt.Println()` calls to use `log.Println()` for consistency with the rest of the autotag codebase.

Reply:
Ah yes, it seems like it does. I'll update it then

Reply:
I adjusted it to use `log.Println`. A few lines up, it runs `log.SetOutput(ioutil.Discard)`, so in order to see any output (which I assume you'd like to see, if it errors out), I manually set log output to ioutil.Stderr before printing the error messages. LMK what you think.